### PR TITLE
Experiment: redesign /new and /thanks pages (Fixes #7139, #7129, #7140)

### LINKED ISSUE #7139: /thanks for Trailhead
## Description

Code the /thanks page for the Trailhead release in June.
[Copy doc](https://docs.google.com/document/d/1F-hIja0JTQTHEK0_zKEFL8vRBBfYK4-LdqzaS-hSvN4/edit)

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [x] /thanks pages are converted to Protocol
- [ ] /thanks pages are done by 5/21

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] This is an example of a risk. Use a checklist!
- [ ] This is an example of a risk.

#### Tasks
- [x] Code the page with copy and design assets provided by Aleks and JKropp

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds experiment to A/B test redesigned /new and /thanks pages ahead of project trailhead launch date.

## Issue / Bugzilla link
#7139
#7129
#7140

## Testing
Demo links:

- https://www-demo1.allizom.org/en-US/firefox/new/?xv=th&v=a (control)
- https://www-demo1.allizom.org/en-US/firefox/download/thanks/?xv=th&v=a (control)
- https://www-demo1.allizom.org/en-US/firefox/new/?xv=th&v=b (redesign)
- https://www-demo1.allizom.org/en-US/firefox/download/thanks/?xv=th&v=b (redesign)

Things to check:

- [ ] Experiment should trigger correctly (en-US, non-Firefox, Windows only)
- [ ] Redesigned pages should look good in all modern browsers, and still be legible/functional in older browsers.
- [ ] Download should still trigger as expected.
- [ ] Other platforms modal should still work as expected.

Successful integration test run: https://ci.vpn1.moz.works/blue/organizations/jenkins/bedrock_multibranch_pipeline/detail/run-integration-tests/304/pipeline

### CODE REVIEW

**Comment by alexgibson** [comment]
There's an ask that this experiment goes live on Monday, so marking as ready for code review.

We're still waiting on final copy.

**Comment by alexgibson** [comment]
Final copy and design is now in place.

**Comment by craigcook** [comment]
Jennifer pointed out that the screenshot isn't rendering very cleanly but I think that's a problem with the original graphic and/or how it was exported from Sketch, so I think they're going to try to provide a cleaner SVG. It wasn't deemed a blocker for the experiment though.

image

The borders around some elements are being lost because it's a really thin, transparent shadow and the anti-aliasing isn't coping with it. Chrome does a slightly better job with the smoothing but it really just needs to be fixed in the original (maybe thicker borders, or none at all).

**Comment by alexgibson** [comment]
This is now fixed with the updated graphic Jennifer provided, thanks.

File: media/js/firefox/new/trailhead/experiment.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var isFirefox = /\sFirefox/.test(navigator.userAgent);
+
+    if (window.site.platform !== 'windows' && !isFirefox) {
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply media/js/firefox/new/trailhead/experiment.js]: The experiment is Windows, non-Firefox users only.

**Comment by alexgibson** [comment]
Thanks @craigcook - nice catch on the experiment targeting. I updated to:

```javascript
if (window.site.platform !== 'windows' || isFirefox) {
    return;
}
```

**Comment by craigcook** [comment]
👍 New graphic looks good, too.

**Comment by alexgibson** [comment]
I'm going to merge this, and if there are any analytics changes required from AP, we can handle those in a followup PR.

**Pull request merged**